src/components/customers/list/CustomersList.js

```diff
@@ -51,11 +53,16 @@ class CustomersList extends Component {
 		const customers = await this.getAllCustomers()
 
 		this.setState({ customers })
-	}
+  }
+  
+  newCustomerClicked() {
+    this.props.history.push("/customers/new")
+  }
 
 	render() {
 		return (
 			<div className="customers-list-wrapper">
+        <Button variant="success" className="new-customer-button" onClick={() => this.newCustomerClicked()}>New Customer</Button>
```

Comment:
variant="secondary"

Reply:
Are you using secondary variant in your products/orders? I prefer the green button for adding stuff, but I can change it to keep it consistent.

Reply:
we should keep it consistent... maybe we can come up with some rules, like:
* use primary for buttons that will do some specific actions, like submit
* use secondary for buttons that do some actions that won't affect data, like - open a modal, open a new page, back button etc.
* use success - maybe we can change variant from primary to success on submit buttons when api call or something has succeeded?

just a quick idea, but we can discuss that

Reply:
so
- success - submitting data
- danger - removing data
- & secondary for everything else?

Reply:
no no, primary for submitting, success after it has been submitted successfully

Reply:
> no no, primary for submitting, success after it has been submitted successfully

I think this is just bad naming in bootstrap, success button does not make much sense to me, you usually get redirected after success or the button can be used again, in which case it should not change colour.

Reply:
hmmmm I see what you mean, well then if you guys are happy with using success for submit buttons then I'm happy for it too. We can use then primary for stuff like open modals etc and secondary for back buttons etc

Reply:
so the very last decision is: (?)
- success = submitting data
- primary = opening modals etc.
- danger = removing data
- secondary = going back etc.

Reply:
Yeah, I'm gonna put it on Wiki